### PR TITLE
Select Collection & Send empty Email if no paper

### PR DESCRIPTION
**Pull request by phhandong**
- Select collection you want via string
- Send empty email if no paper found while nothing go wrong

### CODE REVIEW

File: .github/workflows/main.yml [hunk 1]
```diff
@@ -8,29 +8,32 @@ jobs:
...
+        uses: astral-sh/setup-uv@v3
+        with:
+          version: '0.5.4'
+
+      - name: Run script
+        env:
+          ZOTERO_ID: ${{ secrets.ZOTERO_ID }}
+          ZOTERO_KEY: ${{ secrets.ZOTERO_KEY }}
+          ZOTERO_TAG: ${{ vars.ZOTERO_TAG }}
+          ZOTERO_TAG: ${{ vars.ZOTERO_TAG }}
```
Comment (TideDra) [inline .github/workflows/main.yml:24]: Are these two variables duplicated?

File: main.py [hunk 2]
```diff
@@ -49,6 +49,29 @@ def filter_corpus(corpus:list[dict], pattern:str) -> list[dict]:
     os.remove(filename)
     return new_corpus
 
+def select_corpus(corpus:list[dict], tags: str) -> list[dict]:
+    tag = tags.split(',')
+    new_corpus = []
+    for c in corpus:
+      for p in c['paths']:
+        if p in tag:
```
Comment (TideDra) [inline main.py:57]: Are these two functions duplicated?

**Review by TideDra — COMMENTED** [review]
Thanks for your contributions! I think sending empty email is a good idea. But if you want to select only a few zotero papers into the corpus, ZOTERO_IGNORE is enough. For example, if only 'LLM/survey' is wanted, you can set the ZOTERO_IGNORE as "* \n !LLM/survey", where `!` is used as a NOT operation.

**Comment by phhandong** [comment]
Hi, I have made the requested changes and updated code. Please have a look when you have time. Thanks!

**Pull request merged**